doc/manual/UserManual.tex

```diff
@@ -520,7 +520,7 @@ \section{splashtools}
 Features include:
 \begin{itemize}
 	\item List all file entries.
-	\item Transparently delete timesteps in all HDF5 files belonging to a single run.
+	\item Transparently delete iteration steps in all HDF5 files belonging to a single run.
```

Comment:
Transparently delete all iterations in all HDF5 files belonging to a single run.